MediaBrowser.Model/Services/QueryParamCollection.cs

```diff
@@ -5,19 +5,11 @@
 
 namespace MediaBrowser.Model.Services
 {
+    // Remove this garbage class, it's just a bastard copy of NameValueCollection
```

Comment:
what is the reason for keeping this "bastard copy" around?

Reply:
The functions that parse formdata use it a lot and I think we should rewrite those functions separately.

Reply:
Okay, so a cleanup for further PRs. I accept the idea.